clients/fides-js/src/lib/consent-types.ts

```diff
@@ -343,8 +343,14 @@ export type PrivacyNotice = {
   privacy_notice_history_id: string;
   cookies: Array<Cookies>;
   default_preference: UserConsentPreference;
+};
+
+// This type is exclusively used on front-end
+export type PrivacyNoticeExtended = PrivacyNotice & {
```

Comment:
This is a key part of the changes. We extend base PrivacyNotice with additional helper props for FE

Reply:
I like the name you had in your PR description more: `PrivacyNoticeWithPreference`